### DISCUSSION TITLE
（灌水）为什么比利总是相信他人？

### DISCUSSION THREAD

**Discussion by Aries-Woof** [share121/inter-knot#2908] [其他]
<img src="https://bbs-static.miyoushe.com/static/2024/03/21/fcba628ce15730cf0dbf480ec338f646_3898608417719823301.png" alt="丽都漫步-比利" width="75" >

**Comment by shiliuyexiafeng** [comment]
可能因为他是人机吧（机器人）<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Reply by Aries-Woof** [reply]
because Believe

**Comment by lllliufeng** [comment]
因为相信，是不需要理由哒（bushi

**Comment by dfddtabc** [comment]
因为他善
